src/utils/reputation.ts

```diff
@@ -50,3 +52,11 @@ export const formatReputationChange = (
   const value = adjustConvertedValue(absoluteChange, decimals);
   return getFormattedNumeralValue(value, absoluteChange);
 };
+
+export const getReputationDifference = (
```

Comment:
Nice to have this as a util!

A little question though - should we handle a case where both percentages are `ZeroValue.Zero`, in which case return `ZeroValue.Zero` instead of `ZeroValue.NearZero`?

Reply:
Hey @davecreaser ! If both values are `ZeroValue.Zero` / `"0"`, it will be parsed as numeric type `0` so in the end, the result will still be `0 - 0 = 0` 👌

Reply:
Ahh of course of course, nice explanation! I completely ignored that of course `isNumeric` will return true for `ZeroValue.Zero`